config/src/client.rs

```diff
@@ -106,3 +106,75 @@ impl Default for ConfigurationProxy {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use iroha_config_base::proxy::{Builder, LoadFromDisk};
+    use proptest::prelude::*;
+
+    use super::*;
+    use crate::torii::{uri::DEFAULT_API_URL, DEFAULT_TORII_TELEMETRY_URL};
+
+    const CONFIGURATION_PATH: &str = "../configs/client_cli/config_test.json";
+
+    #[allow(clippy::expect_used)]
+    fn placeholder_keypair() -> KeyPair {
```

Comment:
Another option would be to generate `KeyPair` with seed and seed would be determined by proptest.

Reply:
I'll choose generating from a private key hex, as having that additional `From` impl might be beneficial for other parts of the code in the future.

Reply:
Update: I naively thought that any randomly generated 512-bit hex would properly decompress into a key, which turned out not to be the case. Thus going for generation from seed, but still adding that `From` impl.